movingpandas/trajectory_generalizer.py

```diff
@@ -200,3 +201,50 @@ def _generalize_traj(self, traj, tolerance):
         new_df = traj.df.iloc[keep_rows]
         new_traj = Trajectory(new_df, traj.id)
         return new_traj
+
+
+class TopDownTimeRatioGeneralizer(TrajectoryGeneralizer):
+    """
+    Generalizes using Top-Down Time Ratio algorithm proposed by Meratnia and De By (DOI: 10.1007/978-3-540-24741-8_44).
+
+    tolerance : float
+        Distance tolerance in Kilometers
```

Comment:
To be consistent with other Generalizer implementations, the tolerance should be specified in map units (and meters  if traj.is_latlon).

Reply:
I dont think I completely understand to what unit should I convert the tolerance metric to. Do you mean meters? Distance is used inside the algo so the tolerance needs to be "translated" to some kind of distance eventually.

Reply:
Yes, map units would usually be meters or feet (depending on the CRS).